designs/osc_high_availability/high_availability.md

```diff
@@ -0,0 +1,100 @@
+# OSC High Availability
+- OSC High Availability eliminates from single points of failure which protects from unplanned downtime.
+- Performs regular checks to make sure each OSC Service Node is working properly, and if one fails, switch it out for one that is working.
+- It tries to bring back the failed service/nodes into the cluster and maintains the same number of service nodes.
+    
+#### There are 2 types of failures
+1. Service Stops
+	OSC service may be stopped abnormaly.
```

Comment:
typo abnormally.

Also formatting does not carry over correctly see below. "OSC service..." need to be on own line

![image](https://user-images.githubusercontent.com/7297623/37495429-6f557ab2-286a-11e8-8f8d-fc2b4fa027d4.png)